src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+`https://b.example` + `https://x.example/doge.png` as the key.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-6.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://a.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+Now the user comes back to `https://a.example` but this time the image
+(`https://x.example/doge.png`) is embedded within an iframe. In this case, the
+key is `https://a.example` + `https://a.example` + `https://x.example/doge.png`
+and a cache hit occurs. (Note that when the top-level site and the iframe are
+the same site, the resource cached with the top-level frame can be used.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-4.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+The user is back at `https://a.example` but this time the image is hosted in an
+iframe from `https://c.example`.
+
+In this case, the image is downloaded from the network because there is no
+resource in the cache that matches the key `https://a.example` +
+`https://c.example` + `https://x.example/doge.png`.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-7.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the domain contains a subdomain or a port number? The user visits
+`https://subdomain.a.example`, which embeds an iframe
+(`https://c.example:8080`), which requests the image.
+
+Because the key is created based on "scheme://eTLD+1", subdomains and port
+numbers are ignored, hence a cache hit occurs.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-5.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the iframe is nested multiple times? The user visits
+`https://a.example`, which embeds an iframe (`https://b.example`), which embeds
+yet another iframe (`https://c.example`), which finally requests the image.
+
+Because the key is taken from the top-frame (`https://a.example`) and the
+immediate frame which loads the resource (`https://c.example`), a cache hit
+occurs.
+
+<div class="clearfix"></div>
+
+## FAQs
+
+### As a web developer, are there any action I should take in response to this change?
+
+This is not a breaking change, but it may impose performance considerations for
+some web services.
```

Comment:
This, for example seems like information that should have been in your introduction.

Reply:
I read through the post once again, but I think it's ok from my POV.